docker-compose.yml

```diff
@@ -0,0 +1,24 @@
+version: '2.2'
+services:
+  # srcds
+  csgo:
+    image: sourceservers/csgo:latest
+    ports:
+      - 27015:27015
```

Comment:
We should standardize whether to publishing and forwarding `tcp` of the same port as well for server info to show up under the master server list. IIRC `srcds` games require but I don’t recall `hlds` requiring them same. And also that the port configuration matches that of said games under https://github.com/startersclan/docker-sourceservers/blob/964a2367cf2237b9ec2409fa817f7ec0fd2d94c2/README.md #entrypoint-and-cmd @leojonathanoh

Reply:
@joeltimothyoh 

`srcds` needs both:
- `tcp` is for `rcon`. 
- `udp` is for sourcequery (masterserver) and game traffic.

`hlds` needs only `udp`:
- `udp` is for sourcequery (masterserver), rcon, and game traffic.

Reply:
@joeltimothyoh looking this line, it's actually bugged, because `27015:27015` only publishes `tcp` by default.

Reply:
Yea so we should update the documented command lines to make them expectant and consistent @leojonathanoh

Reply:
> @joeltimothyoh looking this line, it's actually bugged, because `27015:27015` only publishes `tcp` by default. 

That as well yes. If this is going into the docs it has better be battle-tested.

Reply:
@joeltimothyoh added some games